app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,104 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
+
+/* universal */
+
+body {
+  padding-top: 60px;
+}
+
+section {
+  overflow: auto;
+}
+
+textarea {
+  resize: vertical;
+}
+
+.center {
+  text-align: center;
+
+  h1 {
+    margin-bottom: 10px;
+  }
+}
+
+/* typography */
+
```

```suggestion
/* typography */
```
Sai coding convention: trc line commetn có line trống nhưng sau thì ko nha

em self check file này gái hì